MAUI/CircularProgressBar/Appearance.md

```diff
@@ -128,13 +128,13 @@ this.Content = circularProgressBar;
 
 The following properties are used to customize the appearance of the circular progress bar:
 
-* `ProgressRadiusFactor:` Defines the outer radius of the progress indicator.
-* `ProgressThickness:` Defines the thickness of the progress indicator.
-* `TrackRadiusFactor:` Defines the outer radius of the track indicator.
-* `TrackThickness:` Defines the thickness of the track indicator.
-* `ThicknessUnit:` Specifies whether the `ProgressThickness` or `TrackThickness` are defined in pixel or factor.
+* [`ProgressRadiusFactor:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_ProgressRadiusFactor) Defines the outer radius of the progress indicator.
+* [`ProgressThickness:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_ProgressThickness) Defines the thickness of the progress indicator.
+* [`TrackRadiusFactor:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_TrackRadiusFactor) Defines the outer radius of the track indicator.
+* [`TrackThickness:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_TrackThickness) Defines the thickness of the track indicator.
+* [`ThicknessUnit:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_ThicknessUnit) Specifies whether the `ProgressThickness` or `TrackThickness` are defined in pixel or factor.
```

Comment:
API links not included for  `ProgressThickness` or `TrackThickness`

Reply:
done